docker-compose.yml

```
@@ -38,6 +39,17 @@ services:
+  #   container_name: bookmark_frontend
+  #   ports:
+  #     - "3000:3000"
+  #   depends_on:
+  #     - backend
+  #   restart: unless-stopped
+
 volumes:
-  postgres_data:
-      
\ No newline at end of file
+  postgres_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)